metadrive/type.py

```diff
@@ -40,6 +40,7 @@ class MetaDriveType:
     CROSSWALK = "CROSSWALK"
     SPEED_BUMP = "SPEED_BUMP"
     DRIVEWAY = "DRIVEWAY"
+    BARRIER = "BARRIER"
```

Comment:
There is already a type called `TRAFFIC_BARRIER`. Can we just name it `GUARDRAIL`?